packages/retail-ui-extensions/src/components/TextField/TextField.ts

```diff
@@ -1,23 +1,39 @@
 import {createRemoteComponent} from '@remote-ui/core';
 import type {BaseTextFieldProps} from '../shared';
-
+/**
+ * @deprecated
```

Comment:
I think adding deprecation note to component itself should be sufficient as it will go away together with all its props.

Reply:
![Screenshot 2023-07-19 at 10 38 28 AM](https://github.com/Shopify/ui-extensions/assets/29528613/30dd7a8d-cf63-4934-b9ad-edeb90ccb70f)

That would be nice but I dont believe thats the case in Typescript